explorer/src/pages/Issuer/components/Portals/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { t } from "i18next";
+import { ArrowRight } from "lucide-react";
+
+import { Button } from "@/components/Buttons";
+import { EButtonType } from "@/components/Buttons/enum";
+import { HelperIndicator } from "@/components/HelperIndicator";
+
+// TODO: add dynamic data via props
+export const Portals: React.FC = () => {
+  return (
+    <div className="flex flex-col gap-6 md:gap-8">
+      <div className="flex items-center text-2xl md:text-[2rem] font-semibold gap-2">
+        <HelperIndicator type="portal" sizeClass="w-1 h-4 md:w-[0.375rem] h-6" /> Portals
```

Comment:
h-4 or h-6. And I would recommend doing this
`size="big"` and inside make it `type Size = "small" | "big"`